pkg/plugins/tainttoleration/taint_toleration.go

```diff
@@ -0,0 +1,171 @@
+package tainttoleration
+
+import (
+	"context"
+	"errors"
+	"reflect"
+	"sort"
+	"time"
+
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/selection"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/utils/clock"
+	clusterapiv1 "open-cluster-management.io/api/cluster/v1"
+	clusterapiv1beta1 "open-cluster-management.io/api/cluster/v1beta1"
+	"open-cluster-management.io/placement/pkg/plugins"
+)
+
+var _ plugins.Filter = &TaintToleration{}
+
+var timeDiffs = sort.Float64Slice{}
```

Comment:
why this is a global var?

Reply:
Want time differences to be recorded by this var when validating TolerationSeconds. It seems this will introduce issues if initial it only once as a global var. 
Fix this by only declaring it as a global var and initialing it in Filter().